packages/config/src/configDefaults.ts

```diff
@@ -155,6 +168,22 @@ export const configProps = ({
           config = {};
         }
 
+        if (network === "dashboard") {
+          const { host: configuredHost, port } = configObject.dashboard;
+          const host =
+            configuredHost === "0.0.0.0" ? "localhost" : configuredHost;
+
+          const userOverrides = config;
+
+          config = {
+            network_id: "*",
+            networkCheckTimeout: 120000,
+            ...userOverrides,
+            url: `http://${host}:${port}/rpc`,
+            skipDryRun: true
+          };
+        }
+
```

Comment:
Yeah after having implemented it in a few different spots we settled on this being the "least bad" place to do it.

Reply:
❓ Do we already have documentation anywhere about the implications of the "test" and "develop" special networks?  It's unlikely that someone would happen to name their network "dashboard" and then hit issues with this, but if we do have any docs about special networks, it may be worth surfacing this new one there. (not blocking)

Reply:
I believe there's no documentation about these special networks. We definitely should address this in our user-facing docs, and we may even want to try and collect knowledge into developer-facing docs (like how+where each of these networks is implemented, known gotchas, etc.)